src/bindings/writeTypescript.ts

```diff
@@ -192,6 +192,20 @@ export function writeTypescript(
     }
 
     // Errors
+    // Map, so user can do something like ExitCodes["Stack Overflow"]
+    w.append(`const ${abi.name}_errorMessages: { [key: string]: number } = {`);
+    w.inIndent(() => {
+        if (abi.errors) {
+            Object.entries(abi.errors).forEach(([k, abiError]) => {
```

Comment:
`for (const [k, abiError] of abi.errors)` would be more readable, concise and performant.

Reply:
@verytactical there is a few of instances of `Object.entries(foo).forEach(([k, v]) ...` in the codebase. Could you please open an issue to refactor those?